packages/nox/src-epetra/NOX_Epetra_FiniteDifference.C

```diff
@@ -408,18 +408,30 @@ bool FiniteDifference::computeJacobian(const Epetra_Vector& x, Epetra_Operator&
 
   double eta = 0.0;  // Value to perturb the solution vector
 
+#ifndef EPETRA_NO_32BIT_GLOBAL_INDICES
   int min = map.MinAllGID();  // Minimum Global ID value
   int max = map.MaxAllGID();  // Maximum Global ID value
   int myMin = map.MinMyGID(); // Minimum Local ID value
   int myMax = map.MaxMyGID(); // Maximum Local ID value
+#else
+  int min = map.MinAllGID64();  // Minimum Global ID value
+  int max = map.MaxAllGID64();  // Maximum Global ID value
+  int myMin = map.MinMyGID64(); // Minimum Local ID value
+  int myMax = map.MaxMyGID64(); // Maximum Local ID value
+#endif
```

Comment:
Shouldn't this else block be long long types instead of int?

Reply:
Good catch Roger.  I fixed this in 27f4a10.